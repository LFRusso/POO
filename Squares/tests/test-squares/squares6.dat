Square0 -2.08407 0.740486 1.24736
Square1 -1.49988 1.048 2.28879
Square2 -0.460408 -0.937049 3.49746
Square3 -0.896982 -1.91362 1.44538
Square4 -0.483739 0.849012 2.7181
Square5 0.662629 -1.13666 0.26947
Square6 -2.18829 1.18516 0.706609
Square7 -0.923743 1.23394 4.63993
Square8 -1.93389 -0.144149 0.648639
Square9 -0.541354 0.0377746 1.96526
Square10 0.913116 -2.02921 4.31598
Square11 2.88988 -0.097178 2.58608
