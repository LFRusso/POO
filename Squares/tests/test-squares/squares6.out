Square0 intercepts Square1 with area 0.623278
Square0 intercepts Square6 with area 0.425653
Square0 intercepts Square7 with area 0.0656147
Square1 intercepts Square2 with area 1.88948
Square1 intercepts Square4 with area 2.91283
Square1 intercepts Square6 with area 0.0128596
Square1 intercepts Square7 with area 3.60145
Square1 intercepts Square9 with area 1.27045
Square2 intercepts Square3 with area 0.472937
Square2 intercepts Square4 with area 4.61182
Square2 intercepts Square5 with area 0.0188249
Square2 intercepts Square7 with area 4.63928
Square2 intercepts Square9 with area 3.70317
Square2 intercepts Square10 with area 6.84719
Square2 intercepts Square11 with area 0.380599
Square4 intercepts Square7 with area 6.34179
Square4 intercepts Square9 with area 2.20147
Square4 intercepts Square10 with area 1.89963
Square5 intercepts Square10 with area 0.00511535
Square7 intercepts Square9 with area 1.51147
Square7 intercepts Square10 with area 2.95116
Square7 intercepts Square11 with area 1.03698
Square9 intercepts Square10 with area 1.00384
Square10 intercepts Square11 with area 5.57657
